source/extensions/filters/network/common/redis/BUILD

```diff
@@ -0,0 +1,40 @@
+licenses(["notice"])  # Apache 2
+
+# Redis proxy L4 network filter. Implements consistent hashing and observability for large redis
```

Comment:
Can you delete this comment as I don't think applies here now that it's common code?

Reply:
done